CMakeLists.txt

```diff
@@ -25,6 +25,22 @@ add_definitions(${_PXR_CXX_DEFINITIONS})
 
 find_package(Maya REQUIRED)
 
+if(CMAKE_WANT_UFE_BUILD)
```

Comment:
All the UFE integration work we've done is protected by this cmake configuration flag and the build define. Once 2019 is officially released we can enable this flag for Maya 2019+.